src/shared/alterationFiltering/AnnotationFilteringSettings.ts

```diff
@@ -187,8 +177,6 @@ export function buildDriverAnnotationControlsHandlers(
                 )
                     driverAnnotationSettings.hotspots = true;
 
-                driverAnnotationSettings.cbioportalCount = true;
-                driverAnnotationSettings.cosmicCount = true;
```

Comment:
@gblaih this is where the bug was.  if you clicked the parent checkbox "on" it would set cosmicCount to true and that's why the driver number would/could go up.  because it would start evaluating using cosmic service, which is now defunct.   so, solution was just to fully get rid of this cosmic code since we don't want to use that service anymore.  ditto cbioPortal count.